.github/workflows/tests.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         php-version:
           - "7.4"
           - "8.0"
+          - "8.1"
```

Comment:
I see this pulls in PHPC's master branch (with serialization fixes), so I'll take the green CI to mean all is well.